spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -145,6 +146,52 @@
     end
   end
 
+  context 'when True_ID response does not contain a decision product status' do
+    let(:true_id_response_success_2) { JSON.parse(LexisNexisFixtures.true_id_response_success_2) }
+    describe 'when a True_ID Decision product is not present in the response' do
+      it 'excludes decision_product_status from logging' do
+        decision_product = get_decision_product(true_id_response_success_2)
+        expect(decision_product).not_to be_nil
+        body_no_decision = true_id_response_success_2.tap do |json|
+          json['Products'].delete_if { |products| products['ProductType'] == 'TrueID_Decision' }
+        end.to_json
+
+        decision_product = get_decision_product(true_id_response_success_2)
+        expect(decision_product).to be_nil
+        success_response_no_decision = instance_double(
+          Faraday::Response, status: 200,
+                             body: body_no_decision
+        )
+        response = described_class.new(success_response_no_decision, config)
+
+        expect(response.to_h[:decision_product_status]).to be_nil
+      end
+    end
+
+    describe 'when a True_ID_Decision does not containta status' do
+      it 'excludes decision_product_status from logging' do
+        decision_product = get_decision_product(true_id_response_success_2)
+        expect(decision_product['ProductStatus']).not_to be_nil
```

Comment:
Same comment as above. Could these contexts be combined to do that work of setting it to nil once?

Reply:
remove expect line which is unneccesary ... thanks @soniaconnolly